ankihub/gui/operations/ankihub_sync.py

```diff
@@ -223,6 +223,8 @@ def _on_sync_done(on_done: Callable[[Future], None]) -> None:
     LOGGER.info("Sync with AnkiHub done.")
     config.log_private_config()
 
+    QTimer.singleShot(500, lambda: refresh_anki_ui())
```

Comment:
@abdnh 

I added the delay here in order to prevent the deck browser from fading out:
https://github.com/user-attachments/assets/7ba2fdb6-fa37-4a2f-b19b-f686b62fefcc

With the delay I don't get a fade-out:
https://github.com/user-attachments/assets/2d7f4ce9-3014-4e10-a6a2-30b212d4ee8e

The fade-out happens because the main window is not focused when `refresh_anki_ui()` is called. (I think it's because the progress dialog of the sync didn't close yet)
https://github.com/ankitects/anki/blob/9b5da546be49f37c8d6c286e09c86074b2f0c278/qt/aqt/main.py#L869-L870

Any ideas on how to improve this?

Reply:
I think I found a good solution using the `focus_did_change` hook